packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -429,20 +429,26 @@ class DevFS {
           assetPathsToEvict.add(archivePath);
       }
     });
+    if (generator != null) {
+      // We run generator even if [dirtyEntries] was empty because we want
+      // to keep logic of accepting/rejecting generator's output simple:
+      // we must accept/reject generator's output after every [update] call.
+      // Incremental run with no changes is supposed to be fast (considering
+      // that it is initiated by user key press).
+      final List<String> invalidatedFiles = <String>[];
```

Comment:
Should this be  final var invalidatedFiles = <String>[];

Reply:
I believe the Flutter style guide suggests to always have variable type explicitly spelled out.